src/components/ErrorBoundary.js

```diff
@@ -16,7 +16,8 @@ export default class ErrorBoundary extends React.PureComponent {
     }
 
     render() {
-        if (this.state.error) {
+        // Ignore errors related to Twitter widget.
+        if (this.state.error && !this.state.error.toString().includes('twttr')) {
```

Comment:
Here you're filtering errors by exception message, which is not quite correct, but might work.